Source/JavaScriptCore/ftl/FTLOperations.cpp

```diff
@@ -470,8 +471,8 @@ JSC_DEFINE_NOEXCEPT_JIT_OPERATION(operationMaterializeObjectInOSR, JSCell*, (JSG
         case PhantomClonedArguments: {
             unsigned length = argumentCount - 1;
             JSGlobalObject* globalObject = codeBlock->globalObject();
-            ClonedArguments* result = ClonedArguments::createEmpty(globalObject, globalObject->clonedArgumentsStructure(), callee, length, nullptr);
-            RELEASE_ASSERT(result);
+            ClonedArguments* result = ClonedArguments::createEmpty(vm, nullptr, globalObject->clonedArgumentsStructure(), callee, length, nullptr);
```

Comment:
Maybe we can a comment here for passing a nullptr global object when we don't want it to throw a OOM error.

Reply:
Name it to nullOrGlobalObjectForOOM.